cmd/flags_core.go

```diff
@@ -73,7 +73,7 @@ func validateAddress(address string) (string, string, error) {
 		return "", "", fmt.Errorf("both protocol and host must present in the address")
 	}
 
-	if _, port, err := net.SplitHostPort(u.Host); err != nil || port == "" {
```

Comment:
Honestly, this is not part of this PR, especially if you are going to squash it into one commit.